lib/components/user/phone-number-editor.tsx

```diff
@@ -188,100 +108,49 @@ class PhoneNumberEditor extends Component<Props, State> {
+    const hasSubmittedNumber = !isBlank(submittedNumber)
+    const isPending = hasSubmittedNumber || this._isPhoneNumberPending()
 
     return (
       <>
         {isEditing ? (
-          // FIXME: If removing the Save/Cancel buttons on the account screen,
-          // make this <FormGroup> a <form> and remove onKeyDown handler.
-          <FormGroup validationState={phoneErrorState}>
-            <ControlLabel>
-              <FormattedMessage id="components.PhoneNumberEditor.prompt" />
-            </ControlLabel>
-            <ControlStrip>
-              <InlinePhoneInput
-                className="form-control"
-                country={phoneFormatOptions.countryCode}
-                onChange={this._handleNewPhoneNumberChange}
-                onKeyDown={this._handlePhoneNumberKeyDown}
-                placeholder={intl.formatMessage({
-                  id: 'components.PhoneNumberEditor.placeholder'
-                })}
-                type="tel"
-                value={newPhoneNumber}
-              />
-              <Button
-                bsStyle="primary"
-                disabled={isPhoneInvalid}
-                onClick={this._handleRequestCode}
-              >
-                <FormattedMessage id="components.PhoneNumberEditor.sendVerificationText" />
-              </Button>
-              {
-                // Show cancel button only if a phone number is already recorded.
-                initialPhoneNumber && (
-                  <Button onClick={this._handleCancelEditNumber}>
-                    <FormattedMessage id="common.forms.cancel" />
-                  </Button>
-                )
-              }
-              {showPhoneError && (
-                <HelpBlock>
-                  <FormattedMessage id="components.PhoneNumberEditor.invalidPhone" />
-                </HelpBlock>
-              )}
-            </ControlStrip>
-          </FormGroup>
+          <PhoneChangeForm
+            isSubmitting={hasSubmittedNumber}
+            onCancel={this._handleCancelEditNumber}
+            onSubmit={this._handleRequestCode}
+            phoneFormatOptions={phoneFormatOptions}
+            showCancel={!!initialPhoneNumber}
+          />
         ) : (
           <FormGroup>
-            <ControlLabel>
+            <FakeLabel>
               <FormattedMessage id="components.PhoneNumberEditor.smsDetail" />
-            </ControlLabel>
+            </FakeLabel>
             <ControlStrip>
-              <InlineStatic>
+              <InlineStatic className="form-control-static">
                 <SpanWithSpace margin={0.5}>
                   {formatPhoneNumber(
-                    isSubmitted ? newPhoneNumber : initialPhoneNumber
+                    hasSubmittedNumber ? submittedNumber : initialPhoneNumber
                   )}
                 </SpanWithSpace>
+                {/* Invisible parentheses for no-CSS and screen readers */}
+                <InvisibleA11yLabel> (</InvisibleA11yLabel>
```

Comment:
I'm not sure if I'm crazy about this -- is there any reason this isn't part of the i18n string?

Reply:
It is a separate visual block, so I didn't feel it was absolutely necessary.